dashboard/config/environments/adhoc.rb

```diff
@@ -36,12 +36,6 @@
   # the I18n.default_locale when a translation can not be found).
   config.i18n.fallbacks = true
 
-  # Whether or not to display pretty apps (formerly called blockly).
-  config.pretty_apps = true
-
-  # Whether or not to display pretty shared js assets
-  config.pretty_sharedjs = true
-
```

Comment:
I would say not a big deal in general, but prefer to keep things like this separate when possible :)